PostFreeSurfer/PostFreeSurferPipeline.sh

```diff
@@ -182,7 +182,14 @@ if [ `${FSLDIR}/bin/imtest ${T2wFolder}/T2w` -eq 0 ]; then
 fi
 
 if [ "${RegName}" = "FS" ] ; then
-  logWarn "WARNING: FreeSurfer registration is deprecated in the HCP Pipelines as it results in poorer cross-subject functional and cortical areal alignment relative to MSMSulc. Additionally, FreeSurfer registration results in dramatically higher surface distortion (both isotropic and anisotropic). These things occur because FreeSurfer's registration has too little regularizattion of folding patterns that are imperfectly correlated with function and cortical areas, resulting in overfitting of folding patterns. See Robinson et al 2014, 2018 Neuroimage, and Coalson et al 2018 PNAS for more details."
+    log_Warn "WARNING: FreeSurfer's surface registration (based on cortical folding) is deprecated in the"
```

Comment:
log_Warn already prefixes "WARNING" to every line, this is redundant

Reply:
And as a corollary, if you want to add a couple spaces to lines 2-onward to indent the text a little bit following the 
`<dateTime>; <toolName>: WARNING: `
"prefix", that is fine, but no need to indent by the full width of "WARNING".

Reply:
Changed.